Cargo.toml

```diff
@@ -24,9 +24,9 @@ lto = true
 debug-assertions = false
 
 [features]
-
 std = []
 default = ["std"]
+opaque-keys = []
```

Comment:
I think the feature flag name should give idea about the function, and not internals of implementation (`dynamic-`, or `flexible-`, `runtime-` instead of `opaque`).